app/mailers/user_notifications.rb

```diff
@@ -601,12 +601,19 @@ def send_notification_email(opts)
 
     # tag names
     if opts[:show_tags_in_subject] && post.topic_id
+      max_tags =
```

Comment:
Looks like a good spot for a ternary operator
```ruby
max_tags = SiteSetting.enable_max_tags_per_email_subject ? 
  SiteSetting.max_tags_per_email_subject : 
  SiteSetting.max_tags_per_topic
```

Reply:
Tried to make this one but seems that stree doesn't like it and goes bak to an `if else` statement